README.md

```diff
@@ -109,6 +109,25 @@ class UsersController < ApplicationController
 end
 ```
 
+## Specifying a filename or content_type
+
+If you are willing to add a specific filename to your attachment, or send in a specific content_type for your file, you can use `data:` to attach the base64 data and specify your `filename:` and/or `content_type:` hash keys afterwards.
```

Comment:
why `afterwards`?

Reply:
True, it doesn't need an specific order. Removing.